common/source/docs/common-holybro-kakuteh7.rst

```diff
@@ -0,0 +1,171 @@
+   -  micro USB port
+   -  All UARTS support hardware inversion. SBUS, SmartPort, and other inverted protocols work on any UART without "uninvert hack"
+   -  microSD Card Slot for logging
+   -  AT7456E OSD
+
+
+Pinout
+======
+
+.. image:: ../../../images/holybro-kakuteh7-pinout.jpg
+    :target: ../_images/holybro-kakuteh7-pinout.jpg
+
+UART Mapping
+============
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+   -  SERIAL0 -> USB
+   -  SERIAL1 -> UART1 (Telem1)
+   -  SERIAL2 -> UART2 (Telem2) (internally connected to BlueTooth Module)
+   -  SERIAL3 -> UART3 (GPS)
+   -  SERIAL4 -> UART4
+   -  SERIAL5 -> not available
+   -  SERIAL6 -> UART6 (UART6 RX normally RCinput unless :re:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` = 1)
+   -  SERIAL7 -> UART7 (RX7 only available, normally used for ESC telemetry)
+
+The SERIAL7 port (UART7) is normally for ESC telemetry, and has an R7 pin on
+both of the ESC connectors.
+
+
+Servo Output Mapping
+====================
+
+The PWM outputs are marked M1-M8 in the above pinouts. The corresponding servo outputs are:
+
+    - M1 -> Servo Output 4
+    - M2 -> Servo Output 1
+    - M3 -> Servo Output 2
+    - M4 -> Servo Output 3
+    - M5 -> Servo Output 5
+    - M6 -> Servo Output 6
+    - M7 -> Servo Output 7
+    - M8 -> Servo Output 8
+
+RC Input
+========
+
+The R6 pin, which by default is mapped to a timer input, can be used for all ArduPilot supported receiver protocols, except CRSF which requires a true UART connection. However, bi-directional protocols which include telemetry, such as SRXL2 and FPort, when connected in this manner, will only provide RC without telemetry. 
+
+To allow CRSF and embedded telemetry available in Fport, CRSF, and SRXL2 receivers, the R6 pin can also be configured to be used as true UART RX pin for use with bi-directional systems by setting the :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` to “1” so it becomes the SERIAL6 port's RX input pin.
+
+With this option, :ref:`SERIAL6_PROTOCOL<SERIAL6_PROTOCOL>` must be set to "23", and:
+
+- PPM is not supported.
+
+- SBUS/DSM/SRXL connects to the R6 pin, but SBUS requires that the :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` be set to "3".
+
+- FPort requires connection to T6 and :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` be set to "7".
+
+- CRSF also requires a T6 connection, in addition to R6, and automatically provides telemetry. Set :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` to "0".
+
+- SRXL2 requires a connection to T6 and automatically provides telemetry.  Set :ref:`SERIAL6_OPTIONS<SERIAL6_OPTIONS>` to "4".
+
+Any UART can be used for RC system connections in ArduPilot also, and is compatible with all protocols except PPM. See :ref:`common-rc-systems` for details.
+
+BlueTooth Telemetry
+===================
+ 
+Wireless Mavlink Telemetry is supported using an internal BlueTooth module connected to UART2. Its name is "" and pairing code is "".
```

Comment:
has this been verified? the last time I tried to attach to a pure SpeedyBee module it would not even ask for pairing...only could connect with SpeedyBee app

Reply:
I have inquiry in to HB...they are asking SpeedyBee

Reply:
I am just going by the info on their site, I do not have one to verify.

Reply:
BT module wont work without firmware change...can power, but not connect since GCS apps do not recognize BLE modules...removing from description....